pkg/virtctl/vnc/vnc.go

```diff
@@ -288,20 +290,20 @@ func checkAndRunVNCViewer(doneChan chan struct{}, viewResChan chan error, port i
 }
 
 func tigerVncArgs(port int) (args []string) {
-	args = append(args, fmt.Sprintf(loopbackIPAddressFmt, port))
+	args = append(args, fmt.Sprintf(listenAddressFmt, port))
```

Comment:
If I pass in `0.0.0.0`, would we then pass to the VNC clients that they should connect to `0.0.0.0:5900`? That does not seem right to me. Do we need a validation which ensures that we only allow specifying `0.0.0.0` if `--proxy-only` is selected?

Reply:
I'm actually wondering if there's any use to having that variable there at all for non-proxy connections. Should it just be set to 127.0.0.1?

Reply:
I agree, does not look helpful. Let's just add validation which only allows setting the address when proxy-only is set.

Reply:
Agreed. I'll take a look at that today 👍. After this is merged I really need to find a clean way to keep this proxy open if the user specifies that, it's the very last thing we need in our application of Kubevirt and then setup is complete

Reply:
Well, the next few days/weeks will be me working on that so any insight would be greatly appreciated. I'm a newbie to Go but I'm glad to help

Reply:
Just pushed the check for that